test/e2e/gorillamux/traces.json

```diff
@@ -9,6 +9,12 @@
               "stringValue": "sample-app"
             }
           },
+          {
+            "key": "telemetry.auto.version",
+            "value": {
+              "stringValue": "v0.1.0-alpha"
```

Comment:
This is going to be brittle. Is there anyway we can set this up so we don't need to change this every release?